### ISSUE TITLE
issue(docs): max-lines rules

### ISSUE THREAD

**Issue by valorkin** [eslint/eslint#6516]
**What version of ESLint are you using?** 2.13.1
**What parser (default, Babel-ESLint, etc.) are you using?** default
**What did you do? Please include the actual source code causing the issue.**
`"max-lines": ["error", 300, {"skipComments": true, "skipBlankLines": true}],
**What did you expect to happen?** max-lines rules should work
**What actually happened? Please include the actual, raw output from ESLint.**
eslint reported
```bash
.../eslint-config-valorsoft/.eslintrc.json:
        Configuration for rule "max-lines" is invalid:
        Value "300,[object Object]" has more items than allowed.
```

**Comment by vitorbal** [comment]
thanks @valorkin! Just verified that this is true. The documentation for `skipBlankLines` and `skipComments` have the wrong config example, e.g.:

```javascript
/*eslint max-lines: ["error", 2, {"skipComments": true}]*/
```

Should actually say:

```javascript
/*eslint max-lines: ["error", {"max": 2, "skipComments": true}]*/
```

**Comment by aubergine10** [comment]
I've got that file on my to-do list, will update shortly. Thanks for the report and confirmation.

**Comment by aubergine10** [comment]
PR submitted.
